net/bulk_emulation/st/main.c

```diff
@@ -70,7 +70,7 @@ struct ping_xprt {
 
 struct ping_xprt xprts[1] = {
 	{
-		.px_xprt = &m0_net_bulk_mem_xprt,
+		.px_xprt = (struct m0_net_xprt *)&m0_net_bulk_mem_xprt,
```

Comment:
I think we should change "struct ping_xprt::px_xprt" to const.
and "struct ping_ctx::pc_xprt" to const.

Reply:
incorporated